src/grid/grid.directive.ts

```diff
@@ -0,0 +1,53 @@
+import { Directive, HostBinding, Input, OnInit } from "@angular/core";
+
+@Directive({
+	selector: "[ibmGrid]"
+})
+export class GridDirective {
+	@HostBinding("class.bx--grid") baseClass = true;
+}
+
+@Directive({
+	selector: "[ibmRow]"
+})
+export class RowDirective {
+	@HostBinding("class.bx--row") baseClass = true;
+}
+
+@Directive({
+	selector: "[ibmCol]"
+})
+export class ColumnDirective implements OnInit {
+	@Input() class = "";
+
+	@Input() columnNumbers = {};
+
+	@Input() offsets = {};
+
+	@HostBinding("class") columnClasses: string;
+
+	ngOnInit() {
+		const classNames = [];
+		try {
+			Object.keys(this.columnNumbers).forEach(key => {
+				if (this.columnNumbers[key] === "nobreak") {
+					classNames.push(`bx--col-${key}`);
```

Comment:
What if input values change? User for some reason wants to re-flow the layout.

Since classes are only set once on init, end user would have to refresh the page.

`get`ter, `set`ter combo for `columnClasses` would do the trick.

Reply:
Done